scripts/untilfail.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Looks like a mistake, unless its there for some kind of integration tests.

Reply:
Removed